apps/api/src/app/shared/framework/response.interceptor.ts

```diff
@@ -17,7 +17,7 @@ export class ResponseInterceptor<T> implements NestInterceptor<T, Response<T>> {
     return next.handle().pipe(
       map((data) => {
         // For paginated results that already contain the data wrapper, return the whole object
-        if (data?.data) {
+        if (data?.data && !data?._id) {
```

Comment:
Amazing! Maybe worth adding a small comment here on why it's added like this here.